docs/Reference/Warnings.md

```diff
@@ -22,6 +22,7 @@ Only experienced users should consider disabling warnings.
 | Code | Description | How to solve | Discussion |
 | ---- | ----------- | ------------ | ---------- |
 | **FSTWRN001** | The specified schema for a route is missing. This may indicate the schema is not well specified. | Check the schema for the route. | [#4647](https://github.com/fastify/fastify/pull/4647) |
+| **FSTWRN002** | You are trying to register a plugin with mixing async and callback style. This will return error in `fastify@5`. | Do not mix async and callback style. | [TBD]() |
```

Comment:
```suggestion
| **FSTWRN002** | The %s plugin being registered mixes async and callback styles, which will result in an error in `fastify@5`. | Do not mix async and callback style. | [#5112](https://github.com/fastify/fastify/issues/5112) |
```

Reply:
Shouldn't it be this PR? Other rows of the Discussion column links directly the PRs 
BTW I promise next time I will pay more attention too 🤣